api/docs/docs.go

```diff
@@ -392,6 +392,191 @@ const docTemplate = `{
                 },
             },
         },
+        "/fund_informations": {
+            "get": {
+                tags: ["fund_information"],
+                "description": "fund_informationの一覧を取得",
+                "responses": {
+                    "200": {
+                        "description": "fund_informationの一覧の取得",
+                    }
+                }
+            },
+            "post": {
+                tags: ["fund_information"],
+                "description": "fund_informationの作成",
+                responses: {
+                    "200": {
+                        "description": "create されたfund_informationが返ってくる",
+                    }
+                },
+                "parameters": [
+                    {
+                        "name": "user_id",
+                        "in": "query",
+                        "description": "user_id",
+                        "required": true,
+                        "type": "integer"
+                    },
+                    {
+                        "name": "teacher_id",
+                        "in": "query",
+                        "description": "teacher_id",
```

Comment:
```suggestion
                        "description": "teacher_id",
                        "required": true,
```
DB見ると、`teacher_id`もnot nullなので、`required`入れてよさそうですね